docs/essentials/toolbars-and-globals.mdx

```diff
@@ -77,22 +79,27 @@ We have a `global` implemented. Let's wire it up! We can consume our new `theme`
   {/* prettier-ignore-end */}
 </IfRenderer>
 
-
 ## Setting globals on a story
 
-By default, custom toolbars and other globals defined with the `globalTypes` annotation, similar to the one created to set up our custom theme, are consistently applied to every story available. However, if you need to, you can override them at the component or story level to use a custom value for a specific story or subset of stories. This will allow you to test specific scenarios (e.g., custom appearance, behavior) where a particular value should be used. To do this, add a `globals` annotation in your story to enable the selected option.
+<Callout variant="warning" icon="🆕">
+  The ability to set globals on a story or component is available in Storybook 8.3+. Some addons, like [backgrounds](./backgrounds.mdx) and [viewport](./viewport.mdx), have been updated to use the `globals` API when a feature flag is enabled.
```

Comment:
When you're able, can you add a link to the features API page in the `feature flag` so that users know about what flags are required